.travis.yml

```diff
@@ -1,14 +1,27 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
+  - 5.5.9
   - 5.6
   - hhvm
 
+env:
+  - SYMFONY_VERSION=3.0.*@dev
+
+matrix:
+  allow_failures:
+    - php: 7
+  include:
+    - php: 5.5.9
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 7
+      env: SYMFONY_VERSION=3.0.*@dev
```

Comment:
As you use the global env `SYMFONY_VERSION=3.0.*@dev` it's redudant to include them in the matrix.